pkgs/by-name/br/breseq/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  callPackage,
+  nix-update-script,
+  libz,
+  autoconf,
+  automake,
+  libtool,
+  perl,
+  R,
+  bowtie2,
+  which,
+  ghostscript,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "breseq-${version}";
+  version = "0.39.0";
+
+  src = fetchFromGitHub {
+    owner = "barricklab";
+    repo = "breseq";
+    rev = "v${version}";
+    sha256 = "DsDX2oGn7Ex50Wnp1phJjCziCzZIeeZOHriUGJbejsk=";
+  };
+
+  buildInputs = [
+    perl
+    libz
+    autoconf
+    automake
+    libtool
+  ];
+
+  buildPhase = ''
+    ./bootstrap.sh
+    ./configure --prefix=$out/unwrapped
+    make -s
+  '';
+
+  installPhase = ''
```

Comment:
Make sure to surround `installPhase` with `runHook preInstall` and `runHook postInstall`